src/plugins/telemetry_management_section/public/components/telemetry_management_section.tsx

```diff
@@ -249,10 +270,10 @@ export class TelemetryManagementSection extends Component<Props, State> {
             toasts.addSuccess(
               newOptInValue
                 ? i18n.translate('telemetry.optInSuccessOn', {
-                    defaultMessage: 'Usage data collection turned on.',
+                    defaultMessage: 'Sharing Usage with Elastic turned on. Thank you!',
```

Comment:
```suggestion
                    defaultMessage: 'Sharing usage with Elastic is enabled. Thank you.',
```

Reply:
Is "thank you" needed?

Reply:
Not needed. I thought it was friendlier. Happy to remove it :)